app/pages/subscription-hub.tsx

```diff
@@ -0,0 +1,24 @@
+import { useAnchorWallet } from '@solana/wallet-adapter-react';
+import { NextPage } from 'next';
+import { useRouter } from 'next/router';
+import { useUser } from '../hooks/userUser';
+
+const SubscriptionHub: NextPage = () => {
+  const router = useRouter();
+  const connectedWallet = useAnchorWallet();
+  const { user } = useUser();
+
+  // Reroute to `/` if wallet not connected.
+  !connectedWallet && router.push('/');
+
+  // Reroute to `/creator-hub` if the user is a creator.
+  user.creatorAccount && router.push('/creator-hub');
```

Comment:
A creator may also be a subscriber, so they'll want to be able to look at this page that shows all of their subscriptions

Reply:
Yes makes sense